packages/framework/src/Migrations/Version20250516104642.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Shopsys\FrameworkBundle\Migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Override;
+use Shopsys\MigrationBundle\Component\Doctrine\Migrations\AbstractMigration;
+
+class Version20250516104642 extends AbstractMigration
+{
+    /**
+     * @param \Doctrine\DBAL\Schema\Schema $schema
+     */
+    #[Override]
+    public function up(Schema $schema): void
+    {
+        $this->sql('ALTER TABLE administrators ALTER display_only_domain_ids DROP NOT NULL');
+        $this->sql('UPDATE administrators SET display_only_domain_ids = null');
```

Comment:
I'm not sure about this - this will reset the domain filter for every administrator. Is it necessary?